src/tsconfig.json

```diff
@@ -17,22 +17,21 @@
         "noImplicitAny": false,
         "noImplicitReturns": true,
         "noImplicitUseStrict": false,
-        "noFallthroughCasesInSwitch": true
+        "noFallthroughCasesInSwitch": true,
+        "skipLibCheck": true
```

Comment:
Is this needed and why, skipping lib check may lead to issues going forward

Reply:
@VladimirAmiorkov 
libCheck often fails for silly reasons such as if multiple libraries have a type called `Log`.
It also adds unnecessary compilation time and sometimes fails when using newer version of TS in a way that has no impact on actual usage.
All it really does is check that other libs are using types correctly/currently, and really doesn't have any usefulness when consuming them.